gogdl/launch.py

```diff
@@ -19,15 +19,24 @@ def get_flatpak_command(id: str) -> list[str]:
     new_process_command = []
     process_command = ["flatpak", "info", id] 
     if os.path.exists("/.flatpak-info"):
-        spawn_test = subprocess.run(["flatpak-spawn", "--host" "ls"], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        if spawn_test.returncode == 0:
-            new_process_command = ["flatpak-spawn", "--host"]
-            process_command = new_process_command + process_command
+        try:
```

Comment:
I've confirmed that running the code in commit 77ecc22fe1422ff8323e5d251b1e4b4a74219331 fixes the crash but does not use the native dosbox-staging. In order to use dosbox-staging on my Steam Deck (i.e., with `flatpak-spawn`), the `os.chdir` line I suggested is needed.

Reply:
I just made a change. Though instead of static chdir to `/tmp` its done to game directory. That way if command still fails it suggests permission issue that would prevent the game from running nevertheless

Reply:
Oops. I guess I didn't post my comment. I had pulled your changes and built it again and it was all working. Thank you for the fix.